src/ScriptEditor/ui/ScriptEditorRoot.tsx

```diff
@@ -161,6 +161,28 @@ function Root(props: IProps): React.ReactElement {
     }
   }
 
+  function loadAllServerScripts(): void {
+    let server;
+    if (!currentScript || !(server = GetServer(currentScript.hostname))) {
+      return;
+    }
```

Comment:
```suggestion
    if (!currentScript) {
      return;
    }
    const server = GetServer(currentScript.hostname);
    if (!server) {
      return;
    }
```
I'd prefer to not set variables inside a condition, it makes it harder to reason about their values.